scratchpad/components/todo/bindings/AuthStrategy.ts

```diff
@@ -0,0 +1,17 @@
+import {AUTHENTICATION_STRATEGY} from '@loopback/authentication';
+import {BasicStrategy} from '@passport/basic';
+
+export class AuthStrategy {
+  constructor(
+    private @userRepo() users
+  )
+  name = AUTHENTICATION_STRATEGY; // 'authentication.strategy'
+  value() {
```

Comment:
+1

A code like this should be rejected by `tsc`. I think the code in `scratchpad` is not run via `tsc` nor `tslint`, that's why a construct like this could get through. Once we start moving the code into real acceptance tests, issues like this one should be caught automatically.